src/libaktualizr-c/test/api-test-utils/api-test-utils.cc

```diff
@@ -32,11 +33,18 @@ Config *Get_test_config(const char *storagePath) {
   config->provision.server = serverAddress;
   config->provision.provision_path = "tests/test_data/cred.zip";
 
-  config->storage.path = storagePath;
+  temp_dir = new TemporaryDirectory;
```

Comment:
Can we use a `unique_ptr` here instead of a raw `new` and `delete`?

Reply:
There is no std::make_unique in C++11.

Reply:
True, but we've filled in the gap: https://github.com/advancedtelematic/aktualizr/blob/d5d0c65d7bc24c87c1c40ca9d14747159bf88e17/src/libaktualizr/utilities/utils.h#L169